client/src/components/polls/PollModal.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import axios from 'axios';
+import { makeStyles } from '@material-ui/core/styles';
+import { Button, Modal, TextField } from '@material-ui/core';
+
+export default function PollModal(props) {
+  const classes = useStyles();
+  const [open, setOpen] = React.useState(false);
+  const [title, setTitle] = React.useState('');
+  const [image1, setImage1] = React.useState('');
+  const [image2, setImage2] = React.useState('');
+  const [loading, setLoading] = React.useState(false);
+  const [userId, setUserId] = React.useState('1234');
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleChange = (e, type) => {
+    switch (type) {
+      case 'title':
+        setTitle(e.target.value);
+        break;
+      case 'image1':
+        console.log(e.target.files[0]);
+        setImage1(e.target.files[0]);
+        break;
+      case 'image2':
+        console.log(e.target.files[0]);
```

Comment:
Let's remove the console.logs before we merge

Reply:
Cool cool. I always forget about the console.logs :/